vector-im/element-android
# [Device management] Update the unknown verification status icon (PSG-824)

## Type of change


## Content


Update the UI to handle verification status of other sessions when the current one is not verified.

## Motivation and context


Closes #7327 

## Screenshots / GIFs





<img src="https://user-images.githubusercontent.com/46314705/195788686-3887eda3-ea4f-4c9e-a856-11921a7cb054.png" width=300 /> <img src="https://user-images.githubusercontent.com/46314705/195788702-2596ebe0-98d3-4ce5-9092-4b51a740c1f6.png" width=300 />
<img src="https://user-images.githubusercontent.com/46314705/195788832-76cfc5a2-19e2-4281-8db8-921638d21dfe.png" width=300 /> <img src="https://user-images.githubusercontent.com/46314705/195788846-4557450a-253e-4e3f-a8d2-41a31dd3bee2.png" width=300 />

## Tests



- Signin to a new session
- Enable the lab flag for the new session manager
- Go to Settings -> Security & Privacy -> Show all sessions
- Check the unverified sessions section does not show in security recommendations
- Check the other session list items display a correct UI
- Go to the session overview of another session
- Check the UI is correct
- Verify the current session
- Check the unverified sessions section shows in security recommendations if there are any unverified sessions
- Check the other session list items display a correct UI
- Go to the session overview of another session
- Check the UI is correct

## Tested devices

- OS version(s): Android 11

## Checklist
